src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -98,14 +98,6 @@ public SarifErrorListItem CurrentSarifResult
         internal void Register()
         {
             ThreadHelper.ThrowIfNotOnUIThread();
-            // Register this object to listen for IVsUpdateSolutionEvents
-            IVsSolutionBuildManager2 buildManager = ServiceProvider.GlobalProvider.GetService(typeof(SVsSolutionBuildManager)) as IVsSolutionBuildManager2;
```

Comment:
>buildManager [](start = 37, length = 12)

What was this doing and why is it no longer needed?

Reply:
It is needed. Didn't mean to remove it. When this code receives a solution closed event it removes temporary files. I put it back.

---
In reply to: [494584282](https://github.com/microsoft/sarif-visualstudio-extension/pull/202#discussion_r494584282) [](ancestors = 494584282)